invite/src/github.ts

```diff
@@ -62,8 +55,12 @@ export class GitHub {
     issue_number: number,
     comment: string
   ): Promise<void> {
-    // https://octokit.github.io/rest.js/#octokit-routes-issues-create-comment
-    // octokit.issues.createComment({owner, repo, issue_number, body})
+    await this.client.issues.createComment({
```

Comment:
Why would this bot create comments? (genuine q)

Reply:
To update threads that an invite was sent/accepted, or in the event of an error that the invite was not sent because of an issue. Without some kinda feedback in the form of a comment, there's no indication that the bot actually did anything.